specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/examples/Workspace/InferencePool/Group/Endpoint/createOrUpdate.json

```diff
@@ -0,0 +1,130 @@
+{
+  "parameters": {
+      "subscriptionId": "00000000-1111-2222-3333-444444444444",
+      "resourceGroupName": "test-rg",
+      "workspaceName": "my-aml-workspace",
+      "poolName": "string",
+      "groupName": "string",
```

Comment:
We also need the backup group. will it come from tags/properties?

Reply:
Yes we will keep it in tags for now